app/views/search/index.html.erb

```diff
@@ -55,8 +55,14 @@
               <%= l(:label_full_text_search_order_target_score) %>
             </label>
             <label>
-              <%= form.radio_button "order_target", "date", name: "order_target" %>
-              <%= l(:label_full_text_search_order_target_date) %>
+              <!-- TODO: 'date' is deprecated but maintained for backward compatibility. -->
```

Comment:
Could you use ERB comment instead of HTML comment because this comment will be meaningless in rendered HTML?

```suggestion
              <!-- 'date' is deprecated but maintained for backward compatibility. -->
```

Reply:
fix: [cbf714b](https://github.com/clear-code/redmine_full_text_search/pull/119/commits/cbf714b93864d15cab86c7d01761a15e4647b63f)